files/kubelet-config.json

```diff
@@ -2,6 +2,8 @@
   "kind": "KubeletConfiguration",
   "apiVersion": "kubelet.config.k8s.io/v1beta1",
   "address": "0.0.0.0",
+  "imageGCHighThresholdPercent": 70, 
```

Comment:
Any reason 70% and not 90%?

Reply:
thats GC kick in at 70%, i believe 90% its bit too much to wait.